x/staking/client/rest/grpc_query_test.go

```diff
@@ -422,11 +422,9 @@ func (s *IntegrationTestSuite) TestQueryDelegationsResponseCode() {
 	s.Require().NoError(err)
 	newAddr := sdk.AccAddress(info.GetPubKey().Address())
 
-	s.T().Log("expect 404 error for address without delegations")
-	res, statusCode, err := getRequest(fmt.Sprintf("%s/cosmos/staking/v1beta1/delegations/%s", val.APIAddress, newAddr.String()))
+	_, statusCode, err := getRequest(fmt.Sprintf("%s/cosmos/staking/v1beta1/delegations/%s", val.APIAddress, newAddr.String()))
```

Comment:
Now we don't need a separate test for `address without delegations`. Let's add this test case to `TestQueryDelegatorDelegationsGRPC`.